packages/git/git.2.0.0/opam

```diff
@@ -49,6 +49,9 @@ depends: [
   "tls"      {with-test}
   "mtime"    {with-test & >= "1.0.0"}
 ]
+conflicts: [
+  "digestif" {>= "1.0.0"}
```

Comment:
Why not just add an upper bound `& < "1.0.0"` to the digestif dependency above instead?

Reply:
Similarly for the other git packages

Reply:
I got a time out from OPAM with such constraint :( I'm not sur why but it seems better to put the constraint as is.

Reply:
Oh, weird. Let me ping @kit-ty-kate

Reply:
Looks like it is still timing out in some cases though

Reply:
I'd consider `& < "1.0.0"` to be the more streamline solution. There are solver timeouts on various containers & old OCaml versions, or does it happen with a current (4.10+) system (e.g. your laptop)? If not, I'd be in favour of no conflicts but upper bounds instead.

Reply:
I can retry with `& < "1.0.0"` but I'm worried about time-out :+1:.